src/main/java/org/apache/commons/lang3/RandomStringUtils.java

```diff
@@ -34,7 +34,11 @@
  * RandomStringGenerator</a> instead.</p>
  *
  * <p>Caveat: Instances of {@link Random}, upon which the implementation of this
- * class relies, are not cryptographically secure.</p>
+ * class relies, are <b>not cryptographically secure</b>.
+ * Do not use this classes' default implementation of {@link Random} in security sensitive locations,
```

Comment:
I think it should be 

this class' default..

Or

These classes'

Or simply

Avoid using the default implementation in this classs if you need a cryptographically secure...